### PR TITLE
Add datagovuk-tech-docs to integration

### PR DESCRIPTION
**Pull request by nimalank7**
Description:
- As part of the migration work away from GOV.UK PaaS to EKS we are moving the datagovuk-tech-docs app
- Add a Helm chart for the integration environment

### CODE REVIEW

**Comment by sengi** [comment]
Are we able to run this as a plain old static site generator on GitHub Actions like the other techdocs-based sites? Would save a whole bunch of toil if so.

File: charts/app-of-apps/values.yaml [hunk 1]
```diff
@@ -14,3 +14,5 @@ datagovukHelmValues:
 
   opensearch:
     replicaCount: 1
+  techDocs:
+    replicaCount: 1
```
Comment (kentsanggds) [inline charts/app-of-apps/values.yaml:18]: replicaCount should probably be 0 as default for Staging and Prod environments

Comment (kentsanggds) [reply charts/app-of-apps/values.yaml:18]: also would be good to have a line space between opensearch and techdocs

File: charts/app-of-apps/values-integration.yaml [hunk 2]
```diff
@@ -88,6 +88,15 @@ datagovukHelmValues:
         alb.ingress.kubernetes.io/ssl-redirect: "443"
       tls:
         enabled: true
+  techdocs:
+    replicaCount: 1
+    ingress:
+      enabled: true
+      host: datagovuk-docs.eks.integration.govuk.digital
+      ingressClassName: ""
+      annotations:
```
Comment (kentsanggds) [inline charts/app-of-apps/values-integration.yaml:97]: needs the alb settings, like Find app above

**Comment by kentsanggds** [comment]
@sengi yes, we can certainly explore that too, which ever way is easiest to deploy

**Comment by sengi** [comment]
Awesome, thanks! Not having another app to manage is a big plus for all of us 👍

**Comment by kentsanggds** [comment]
No longer needed as using github pages for hosting

**Pull request closed without merging**